cloud-sql/mysql/sqlalchemy/main.py

```diff
@@ -68,39 +61,51 @@ def init_connection_engine():
 
 def init_tcp_connection_engine(db_config):
     # [START cloud_sql_mysql_sqlalchemy_create_tcp]
-    db_socket_addr = os.environ.get("DB_HOST").split(":")
+    # Remember - storing secrets in plaintext is potentially unsafe. Consider using
+    # something like https://cloud.google.com/kms/ to help keep secrets secret.
+    db_user = os.environ.get("DB_USER")
+    db_pass = os.environ.get("DB_PASS")
+    db_name = os.environ.get("DB_NAME")
+    db_host = os.environ.get("DB_HOST")
+
+    db_host_parts = db_host.split(":")
 
-    # Extract host and port from socket address
-    db_host = db_socket_addr[0]
-    db_port = int(db_socket_addr[1])
+    # Extract host and port from db_host socket address
+    db_hostname = db_host_parts[0]
+    db_port = int(db_host_parts[1])
```

Comment:
Can we condense this a bit?
```suggestion
    # Extract host and port from db_host socket address
    host_args = db_host.split(":")
    db_hostname, db_port = host_args[0], int(host_args[1])
```